app/frontend/src/styles/application/signin.scss

```diff
@@ -10,3 +10,48 @@
     color: govuk-colour('white');
   }
 }
+
+.job-alert-signin {
+  @include govuk-responsive-padding(5);
+  margin-bottom: govuk-spacing(5);
+
+  &__submit {
+    width: 100%
+  }
+
+  &__prompt {
+    text-align: center;
+  }
+
+  .wrap-email {
+    word-break: break-all;
+  }
+
+  &--blue {
+    background-color: govuk-colour('blue');
+
+    .govuk-link:not(:focus) {
```

Comment:
does this do anything when line 37 follows?

Reply:
i guess so link isnt blue when it has :visited psuedo class, am i correct? if so 👍

Reply:
(or the purple colour i think it is for gov links that would look very similar to background)

Reply:
Without it the link is blue. 

<img width="436" alt="Screenshot 2021-05-14 at 13 43 45" src="https://user-images.githubusercontent.com/30624173/118272442-8a2c2e00-b4ba-11eb-87b1-7981f4d8b5b8.png">

I don't think this has anything to do with a :visited psuedo class? It seems, for some reason that the style from the govuk-components gem (in _links.scss I think?) appears to override the style from signing.scss

Is that correct and if so, why is this / could this be the case?

Reply:
hmmm ok, if thats the case and the * rule below does make it white its because the rule isnt specific enough, if it was it would overide it (remember the specificty order we went through many months ago in prep for your exam. if its the case its a slight misuse of this selector but it is making it more specific. it doesnt matter too much for the purposes of this abtest, just letting you know. i have a closer look when i finished my lunch if theres a obvious alternative. and if you want me to explain what i meant about visited i can as that was the only proper reason i could think of to use it - i say 'proper' theres no right or wrong, but you get to pick up on possible little code smells like this...

Reply:
dont neccessarily need to change this for this PR, just pointing it out